addons/mrp/views/stock_rule_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_stock_rule_form_inherit_mrp" model="ir.ui.view">
```

Comment:
We should [use the same ID than inherited view](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#inheriting-xml) (it will be `mrp.view_stock_rule_form` so no conflict with `stock.view_stock_rule_form`)
```suggestion
        <record id="view_stock_rule_form" model="ir.ui.view">
```

Reply:
Good catch, thanks the reminder! :slightly_smiling_face: